docs/components/Search.md

```diff
@@ -133,7 +133,7 @@ To set up the scroll-away mode, use a top-level `CoordinatorLayout` and place
 the `SearchBar` within an `AppBarLayout`. Then, place the `AppBarLayout` below
 the scrolling view (usually a `RecyclerView` or `NestedScrollView`) in the
 `CoordinatorLayout`, and set
-`app:layout_behavior="@string/searchbar_scrolling_view_behavior"` on the
+`app:layout_behavior="@string/appbar_scrolling_view_behavior"` on the
```

Comment:
https://github.com/material-components/material-components-android/issues/3358#issuecomment-1817764600
Please tell me the correct way.

Reply:
I mentioned in https://github.com/material-components/material-components-android/issues/3103#issuecomment-1572125339 that the correct way is probably to use `app:statusBarForeground="?attr/colorSurface"` on the AppBarLayout so that there's some "protection" for the status bar content on scroll.

I've tested this locally and it works well and matches the expectation of the Material spec:
- https://m3.material.io/components/search/guidelines#b340d738-9c8a-4258-876e-b4ac892616c1

So I will update the demos and the doc to follow that suggestion.

Reply:
app:statusBarForeground="?attr/colorSurface" is tedious but gives a good result.
@string/searchbar_scrolling_view_behavior is still private (https://github.com/material-components/material-components-android/issues/3358#issuecomment-1518022524).

Reply:
Thanks, will make the string public in a separate commit.

Reply:
> The searchbar one is important because it handles AppBarLayout transparency and overlapping with the rest of the content, as mentioned in the following sentence.

Can #3102 be closed, too?

Reply:
Yep!